torch/_inductor/kernel/mm_plus_mm.py

```diff
@@ -93,8 +92,8 @@ def ref_mm_plus_mm(a, b, c, d, out):
         D += BLOCK_K * stride_dk
 
     # rematerialize rm and rn to save registers
-    rm = pid_m * BLOCK_M + tl.arange(0, BLOCK_M)
-    rn = pid_n * BLOCK_N + tl.arange(0, BLOCK_N)
+    #rm = pid_m * BLOCK_M + tl.arange(0, BLOCK_M)
+    #rn = pid_n * BLOCK_N + tl.arange(0, BLOCK_N)
```

Comment:
Is this rematerialization a bad idea now, or is it temporary?  Probably we should either delete it (or drop in a comment describing why it's temporary).

Reply:
Didn't see any difference with or without, deleted